apps/fem/fem2.cpp

```diff
@@ -0,0 +1,139 @@
+#include "graph.h"
```

Comment:
Can we change file names to be more reflective of their function. Like `fem_triangles.cpp`, `fem_tetrahedra.cpp`, `poisson_triangles.sim`, `poisson_tetrahedra.sim`?